en/1-who-we-are.md

```diff
@@ -11,4 +11,4 @@ To build the WoM and Meeds, the association leverages the Ethereum platform and
 
 The association is structured as a Decentralized Autonomous Organization (DAO) where the governance token holders vote on all important organizational changes such as electing the board every year.
 
-Besides Meeds token management, the association holds the Intellectual Property (IP) of the Meeds software [github.com/meeds-io](https://github.com/meeds-io), available under the LGPL license and distributed for free. The latter includes an Ethereum wallet app that users can use to store and transact their Meeds (see section 2.4 for more details). The Meeds Association also runs a multi-tenant cloud hosting service leveraging that software.
+Besides Meeds token management, the association holds the Intellectual Property (IP) of the Meeds Open Source Software [github.com/meeds-io](https://github.com/meeds-io), that powers community Hubs. The Meeds Association also runs a cloud platform to hosts Hubs.
```

Comment:
```suggestion
Besides Meeds token management, the establishment holds the Intellectual Property (IP) of the Meeds Open Source Software [github.com/meeds-io](https://github.com/meeds-io), that powers community Hubs. The Meeds ecosystem  also runs a cloud platform to hosts Hubs.
```

Reply:
I think that the word "association" has not better described what Meeds DAO is and base on the context of that statement, I believe that the word "establishment" will do.
Secondly, I think that the phrase "The meeds Association" is not appropriate for that sentience.